exp/addons/controllers/clusterresourceset_controller_test.go

```diff
@@ -284,6 +357,8 @@ metadata:
 			return false
 		}, timeout).Should(BeTrue())
 
+		time.Sleep(2 * time.Second)
```

Comment:
Same as above (do we really need a delay given the eventually loop down in the code)?